deploy/test/Dockerfile

```diff
@@ -21,8 +21,9 @@ RUN wget -O /usr/local/bin/kubectl ${KUBECTL_CLI_URL:-https://storage.googleapis
 ARG OPENSHIFT_CLI_URL
 RUN mkdir -p ocbin && \
     wget -O oc.tar.gz ${OPENSHIFT_CLI_URL:-https://github.com/openshift/origin/releases/download/v3.7.2/openshift-origin-client-tools-v3.7.2-282e43f-linux-64bit.tar.gz} && \
-    tar xvf oc.tar.gz --strip-components=1 -C ocbin && \
-    mv ocbin/oc /usr/local/bin/oc && \
+    tar xvf oc.tar.gz -C ocbin && \
+    ls -la ocbin && \
```

Comment:
Is this change needed?

Reply:
OCP cli extracts differently between v3.* and v4.*. See: https://github.com/conjurdemos/kubernetes-conjur-demo/pull/144#discussion_r692377626

Reply:
Ah gotcha. This works too:
```
RUN mkdir -p ocbin && \
    wget -O oc.tar.gz ${OPENSHIFT_CLI_URL:-https://github.com/openshift/origin/releases/download/v3.7.2/openshift-origin-client-tools-v3.7.2-282e43f-linux-64bit.tar.gz} && \
    tar xvf oc.tar.gz --transform='s/.*\///' -C ocbin` && \
    mv ocbin/oc /usr/local/bin/oc && \
    rm -rf ocbin oc.tar.gz
```
Don't feel too strongly about it though. In any case the `ls` line can be removed

Reply:
I went ahead and removed the `ls` command. I'm going to leave the rest as is just because it is consistent with the other repo for now. Just let me know if you want me to change it to yours for sure and I can =).